pkg/azure/provider.go

```diff
@@ -265,6 +291,8 @@ func (p *Provider) GetServicePrincipalToken(env *azure.Environment, resource str
 func (p *Provider) MountSecretsStoreObjectContent(ctx context.Context, attrib map[string]string, secrets map[string]string, targetPath string, permission os.FileMode) (err error) {
 	keyvaultName := attrib["keyvaultName"]
 	usePodIdentityStr := attrib["usePodIdentity"]
+	useVMManagedIdentityStr := attrib["useVmManagedIdentity"]
+	userAssignedIdentityIDStr := attrib["userAssignedIdentityID"]
```

Comment:
since this is only used as a string, we can rename this var to `userAssignedIdentityID`?